### DISCUSSION TITLE
LEDs not responding to Local CC multiple value mapping

### DISCUSSION THREAD

**Discussion by hotelbachelard** [shanteacontrols/OpenDeck#267] [General]
Hello, I inherited an older OpenDeck M rev. 1.2.1 board running firmare 4.0.3, and am using it to build a custom MIDI controller for a theatre show where it will be used on stage by actors to control things on an Behringer X32 (mic volume, fx sends, speaker volume, etc.) as well as trigger sound cues in QLab.
I've been testing faders and buttons and switches and everything is working great, and I wanted to see if I can connect some LEDs with some of the faders so that the control panel can have more visual feedback on stage. I tested LEDs with buttons, works perfectly. I was testing various LEDs with a slider (for example, slider connected to Analog 0, MIDI Channel 2, CC #8, standard 0-127 value). I read on the Wiki that newer boards do not have brightness control but I assume that means older boards do. I was not able to control brightness with a Local CC (Multiple Value) assignment on an LED. I've tried multiple LEDs, multiple activation values, etc and I'm not getting a corresponding brightness variation. 

Any insight/help appreciated, thank you.

**Comment by paradajz** [comment]
The v1 board doesn't support brightness control - only the v2 ones do.

**Comment by hotelbachelard** [comment]
Awesome, that's all I needed to know. It wasn't super clear.
